webapp/src/components/reaction_stream/reaction_stream.tsx

```diff
@@ -50,16 +61,48 @@ const ReactionChip = styled.div<chipProps>`
   `}
 `;
 
+type VoiceUserStatuses = {
+    [key: string]: UserState,
+};
+
 // add a list of reactions, on top of that add the hands up as the top element
 export const ReactionStream = (props: Props) => {
-    const reversed = [...props.reactions];
+    const vReactions = useSelector(voiceReactions);
+    const currentUserID = useSelector(getCurrentUserId);
+
+    const statuses = useSelector(voiceUsersStatuses) as VoiceUserStatuses;
+    const vConnectedProfiles = useSelector(voiceConnectedProfiles);
+
+    const cChannelID = useSelector(connectedChannelID);
+    const channel = useSelector((state: GlobalState) => getChannel(state, cChannelID));
+
+    const screenSharingID = useSelector((state: GlobalState) => voiceChannelScreenSharingID(state, channel?.id)) || '';
+
+    const sortedProfiles = (profiles: UserProfile[], sta: {[key: string]: UserState}) => {
+        return [...profiles].sort(alphaSortProfiles(profiles)).sort(stateSortProfiles(profiles, sta, screenSharingID));
+    };
+    const profiles = sortedProfiles(vConnectedProfiles, statuses);
+
+    // building the list here causes a bug tht if a user leaves and recently reacted it will show as blank
+    const profileMap: {[key: string]: UserProfile;} = {};
+    profiles.forEach((profile) => {
+        profileMap[profile.id] = profile;
+    });
+    const handsup: string[] = [];
```

Comment:
Please reflect the change to `handsup` that was made in this PR: https://github.com/mattermost/mattermost-plugin-calls/pull/215
This ensures the hands are shown in the order that they were raised in.